packages/animations/lib/src/open_container.dart

```diff
@@ -16,9 +16,12 @@ typedef CloseContainerActionCallback<S> = void Function({S? returnValue});
 ///
 /// The `action` callback provided to [OpenContainer.openBuilder] can be used
 /// to close the container.
+///
+/// The `extras` is that value shared by [closedBuilder]'s `action`
 typedef OpenContainerBuilder<S> = Widget Function(
   BuildContext context,
   CloseContainerActionCallback<S> action,
+  dynamic extras,
```

Comment:
dynamic basically turns off the type system, which often leads to bugs. We'd like to avoid it in our public API.